frontend/src/metabase/visualizations/visualizations/SmartScalar/SmartScalar.jsx

```diff
@@ -1,62 +1,57 @@
 /* eslint-disable react/prop-types */
```

Comment:
Can we convert this into TypeScript or is there a reason this file was not converted into TypeScript?